lib/assessment.js

```diff
@@ -8,38 +8,74 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
+var min = function min(previousValue, currentValue) {
+  if previousValue < currentValue{
+    return result
+  } else {
+    return currentValue
+  }
 };
+var result = [3,7,1,10].reduce(min);
+
+console.log(result);
+
+// Source used https://msdn.microsoft.com/library/ff679975(v=vs.94).aspx
 
 // Question 2
 // Write a callback for reduce that produces
 // the sum of the values in the array
-var sum = function sum() {
+var reduce = function reduce(array, callback){
 
+var sum = function sum(previousValue, currentValue) {
+  return previousValue + currentValue;
```

Comment:
This (lines 29-31) is a good a correct answer. Lines 27 and 34 wrap it in a function and make it inaccessible from outside. Line 36, then, can't work, and neither can the tests written for this assessment. 3/4